hub/src/SshKeysController.cpp

```diff
@@ -149,19 +153,27 @@ void CSshKeysController::generate_new_ssh_key(QWidget *parent) {
   QFile key(str_private);
   QFile key_pub(str_public);
 
-  if (key.exists() && key_pub.exists()) {
-    key.remove();
-    key_pub.remove();
-  }
+  if (fi.baseName().contains(QRegularExpression("[/|\\\\$%~\"*?:<>^]")) ||
```

Comment:
`\\\\` ?

Reply:
in QRegularExpression \ should be defined as \\\\